api/orders/index.js

```diff
@@ -64,6 +64,12 @@
             }
             console.log("Received response: " + JSON.stringify(body));
             var jsonBody = JSON.parse(body);
+
+            if (jsonBody._links == undefined) {
+              callback(error);
```

Comment:
I'm closing this issue, since #49 solves just this problem. I'll investigate the nodejs crashes tomorrow.